### PR TITLE
fix: If installing a theme from Disco Pane, the description will not be displayed in more section for that theme

### PR DESCRIPTION
**Pull request by aayushsanghavi**
Fixes: #1572 

Added description key to themeData and modified the corresponding test

### CODE REVIEW

**Comment by aayushsanghavi** [comment]
@tofumatt please have a look at the changes

File: src/core/themePreview.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ export default function themeAction(node, action, _doc = document) {
   node.dispatchEvent(event);
 }
 
-export function getThemeData({ id, name, headerURL, footerURL, textcolor, accentcolor, author }) {
+export function getThemeData({ id, name, description, headerURL, footerURL, textcolor, accentcolor, author }) {
```
Comment (tofumatt) [inline src/core/themePreview.js]: This is failing the lint test, the fix would be:

```js
export function getThemeData({
  id, name, description, headerURL, footerURL, textcolor, accentcolor, author,
}) {
```

**Comment by aayushsanghavi** [comment]
Done! @tofumatt thanks for reviewing!

File: src/core/themePreview.js [hunk 2]
```diff
@@ -11,7 +11,9 @@ export default function themeAction(node, action, _doc = document) {
   node.dispatchEvent(event);
 }
 
-export function getThemeData({ id, name, headerURL, footerURL, textcolor, accentcolor, author }) {
+export function getThemeData({
+id, name, description, headerURL, footerURL, textcolor, accentcolor, author,
```
Comment (tofumatt) [inline src/core/themePreview.js]: Oh, sorry, can you just indent this? Sorry the linter didn't catch it but that's our usual style for this kind of argument setup.

**Comment by tofumatt** [comment]
Thanks! 👍

**Comment by aayushsanghavi** [comment]
Thanks for the merge @tofumatt!

**Pull request merged**